pyproject.toml

```diff
@@ -8,20 +8,65 @@ build-backend = "scikit_build_core.build"
 [project]
 name = "halide"
 authors = [{ name = "The Halide team", email = "halide-dev@lists.csail.mit.edu" }]
+maintainers = [{ name = "Alex Reinking", email = "areinking@adobe.com" }]
```

Comment:
Let's open an issue to add a virtual email address here that will go to multiple people, in case you are hit by a bus